### PR TITLE
Raw angle signal data for easy checksum calc, and one less gear bit

### PR DESCRIPTION
**Pull request by Jafaral**
Tidy up a few signals for the the Mazda port PR

### CODE REVIEW

File: mazda_cx5_gt_2017.dbc [hunk 1]
```diff
@@ -491,8 +491,10 @@ BO_ 552 GEAR: 8 XXX
  SG_ NEW_SIGNAL_5 : 26|3@0+ (1,0) [0|255] "" XXX
  SG_ NEW_SIGNAL_6 : 31|5@0+ (1,0) [0|31] "" XXX
  SG_ NEW_SIGNAL_7 : 39|1@0+ (1,0) [0|255] "" XXX
- SG_ GEAR : 35|4@0+ (1,0) [0|15] "" XXX
  SG_ MORE_GEAR : 7|4@0+ (1,0) [0|255] "" XXX
+ SG_ GEAR : 35|3@0+ (1,0) [0|15] "" XXX
+ SG_ NEW_SIGNAL_1 : 0|3@1+ (1,0) [0|3] "" XXX
```
Comment (quillford) [inline mazda_cx5_gt_2017.dbc:496]: don't need to define unknown signals

Comment (Jafaral) [reply mazda_cx5_gt_2017.dbc:496]: want me do drop that bit ?

Comment (quillford) [reply mazda_cx5_gt_2017.dbc:496]: yeah or give it a better name if you know what it is

Comment (Jafaral) [reply mazda_cx5_gt_2017.dbc]: coming your way... with GEAR values updated to allow parsing. Will wait for this PR to trickle in OP before updating the Mazda port PR for anything that depend on it

File: mazda_cx5_gt_2017.dbc [hunk 2]
```diff
@@ -144,8 +144,8 @@ BO_ 579 CAM_LKAS: 8 XXX
  SG_ LDW : 23|1@0+ (1,0) [0|1] "" XXX
  SG_ LKAS_REQUEST : 3|12@0+ (1,-2048) [0|2048] "" XXX
  SG_ ERR_BIT_2 : 30|1@0+ (1,0) [0|1] "" XXX
- SG_ LKAS_ANGLE : 33|12@0+ (0.022,-45.06) [0|1] "" XXX
- SG_ BIT2 : 52|1@0+ (1,0) [0|1] "" XXX
+ SG_ ANGLE_ENABLED : 52|1@0+ (1,0) [0|1] "" XXX
+ SG_ STEERING_ANGLE : 33|12@0+ (1,-2048) [-2048|2047] "" XXX
```
Comment (quillford) [inline mazda_cx5_gt_2017.dbc:148]: Looks like this should be signed

Comment (Jafaral) [reply mazda_cx5_gt_2017.dbc:148]: Why? it is a 12 bit value [0:4095] wirh -2048 offset giving values in the range of [-2048:2047]. signed values are very rare in Mazda, offset is generally used.

**Pull request merged**